### DISCUSSION TITLE
Album view displays empty album contents for albums names which contain a forwardslash ( / ).

### DISCUSSION THREAD

**Discussion by Sagnasty** [rern/rAudio#1493] [Fixed Issues]
Running rAudio 20231111.

Album view will not display album contents for albums names which contain a forward slash ( / ).   Example   **Smiley Smile / Wild Honey**

Album contents are displayed fine in other views (Album artist, artist, etc.).

Album view will display the contents if the album name containing a forward slash does not have a leading and trailing space.  Example  
**The Best of David Bowie 1974/1979**

Jim

3 x Raspberry Pi 3B+ Rev 1.3 > HiFiBerry DAC+ Pro
1 x Raspberry Pi 3B Rev 1.2 > HiFiBerry DAC+ Pro
Synology DS718+

**Comment by rern** [comment]
Try editing album tag of an album which doesn't contain slashes. I've just done, including the `/` and `:` in album name, and it ran fine.

**Reply by Sagnasty** [reply]
The tag editor is **not** accessible for all album names which include  **/** and **:** 

Tag editor is accessible and is working fine for all other albums

Some more info:

I did not have this problem with these albums in 20230620.

I can reproduce the same behavior on a Raspberry Pi 3B+ Rev 1.3 and Raspberry Pi 3B Rev 1.2

**Reply by rern** [reply]
Try with accessible ones to proof that album names can contains the `/` and `:` .
By the way, post folder/directory names with full path of the albums.

**Reply by Sagnasty** [reply]
There are probably more than 100 albums affected, so I will chose one and hopefully can provide you with good details.

These views (Album artist, Artist, and Genre) are working fine:
![Artist view](https://github.com/rern/rAudio/assets/76575081/0b68278e-9e84-4236-892d-5c23686fb006) 

![Album artist view](https://github.com/rern/rAudio/assets/76575081/68062d6d-6bd4-4f43-b62f-efa41c243b22)

![Genre view](https://github.com/rern/rAudio/assets/76575081/dc256e98-3eed-496b-b361-053d905ea9cf)

![Tag editor view](https://github.com/rern/rAudio/assets/76575081/7d800090-b7c6-469d-be5a-da83a5b15507)

![File path info](https://github.com/rern/rAudio/assets/76575081/7c003542-0d07-4029-b883-3bda678ac5b8)



The album view details:
![Album view 1](https://github.com/rern/rAudio/assets/76575081/f6a135a7-4ac2-431f-9c23-f1a3ff6cd48c)

![Album view 2](https://github.com/rern/rAudio/assets/76575081/7ccf9dbe-8315-47f9-87cd-d9f282cb5785)

I also confirmed that the cover, cover art, and thumb jpg files are located in the album directory.

**Reply by rern** [reply]
I have renamed a directory/folder to include double spaces and it reproduced the issue.
Please try renaming one of yours by replacing all double/multiple spaces with single spaces and if it fixed the issue.

Next update will support double spaces in directory/folder names.

**Comment by Sagnasty** [comment]
I confirmed that renaming all double/multiple spaces with single spaces (and a library refresh) now correctly shows the album art and album contents in the Album view.

20th Century Masters  The Millennium Collection  The Best of 38 Special    -->    20th Century Masters The Millennium Collection The Best of 38 Special


![image](https://github.com/rern/rAudio/assets/76575081/6ca0f4ac-9807-4a28-ab91-f4355de88ac2)


Thanks very much

**Reply by Sagnasty** [reply]
Also, I just noticed that the latest view is also affected by the double space issue

![Latest view 1](https://github.com/rern/rAudio/assets/76575081/ac64f1f3-b926-475a-a14f-0e98438056a3)

![Latest view 2](https://github.com/rern/rAudio/assets/76575081/7f57f426-4ef0-4eb4-a2f8-a1921bea3895)

**Comment by Sagnasty** [comment]
Confirmed fixed in rAudio 20231118

Note: you must rebuild the entire library database after applying the update.
